src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsWindowMapping.java

```diff
@@ -69,6 +94,67 @@ public static LocalDateTime getFirstDateTimeMustBe() {
         return null;
     }
 
+    @NotNull
+    private static LocalDate handleAdjournedHearingType(SscsCaseData caseData) {
+        String dateType = caseData.getAdjournCaseNextHearingDateType();
+        if (FIRST_AVAILABLE_DATE.equalsIgnoreCase(dateType)) {
+            return LocalDate.now().plusDays(DAYS_TO_ADD_HEARING_WINDOW_TODAY_ADJOURNMENT);
+
+        } else if (FIRST_AVAILABLE_DATE_AFTER.equalsIgnoreCase(dateType)) {
+            return calculateDateOfAdjournedHearing(caseData);
+
```

Comment:
Remove whitespace (check all your code for extra added whitespace)

Reply:
done

Reply:
lines 34, 99, 102, 124, 132 are all extra bits of whitespace

Reply:
done